docs/IntelOwl/contribute.md

```diff
@@ -424,6 +424,107 @@ You can use the django management command `dumpplugin` to automatically create t
 
 Example: `docker exec -ti intelowl_uwsgi python3 manage.py dumpplugin PlaybookConfig <new_analyzer_name>`
 
+## DataModel
+In version XXX of IntelOwl, a new plugin has been added: the `DataModel`.
+Its main functionality is to model an `Analyzer` result to a set of prearranged keys, allowing users to easily search, evaluate and use the analyzer result.
+The author of an `AnalyzerConfig` is able to decide mapping between each field of the `AnalyzerReport` and the corresponding in the `DataModel`.
```

Comment:
The basic description of each Plugin should be put in the "Usage section" so I expect these 3 rows there while I would name this overall section "How to create/customize the Data Model" or something similar to align it with the rest of the existing documentation